src/encoded/static/components/genome_browser.js

```diff
@@ -1,610 +1,501 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'underscore';
+import { FetchedData, Param } from './fetched';
+import { BrowserFeat } from './browserfeat';
 
```

Comment:
This file was copied over from Encode. There have been some small changes but not too many.